supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -27,3 +27,29 @@ Provide the source language if it is supported by the Red Hat Customer Portal to
 == Long code examples in procedures
 
 All code blocks (regardless of length) must be necessary, accurate, and helpful. Code blocks must be as copy-and-paste friendly as possible, with the exception of variables and callouts. The length of the block is irrelevant, within reason, if the code block follows these guidelines.
+
+[[commands-with-elevated-privileges]]
+== Commands with elevated privileges
+
+Some commands require elevated privileges to run. Users without elevated privileges must first do one of the following to run such a command:
+
+* Use `su -i` to switch to the root user account.
+* Preface the command with `sudo` to temporarily change their current privileges.
+
+=== Guidelines for documenting commands with elevated privileges
+
+When documenting commands that require elevated privileges, consider the following guidelines and formulate a consistent approach to use throughout your content:
```

Comment:
One suggestion to be a bit more prescriptive, direct, and minimal. The word 'consider' is a a bit vague and I tend to remove it from our docs so we could maybe do the same here:

`Use the following guidelines when you document commands that require elevated privileges:`

As I tried to remove 'consider' I ended up reducing this lead-in sentence to this ^^ much shorter one, which I think gets the job done. But I'd love to get @bergerhoffer and @IngridT1 's opinions too -have I shaved too close to the bone with this suggestion?

Reply:
I hope you don't mind me butting in! I get the notifications and sometimes I have a little poke around. 

+1 your suggested rewrite @ndeevy. It's correct and concise. 

We don't need to include "formulate a consistent approach to use throughout your content" because as tech writers, we should be consistent anyway.

Reply:
(always welcome @JoanneOFlynn2018)

Reply:
@ndeevy and @JoanneOFlynn2018 Excellent points. I'll make the changes.